### PR TITLE
chore: add contributing guide

### PR DESCRIPTION
**Pull request by malept**
Adds a `CONTRIBUTING.md` to the repository (based on the one I wrote for Electron Packager).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+```
+
+good:
+
+```
+commit 1: add foo option
+commit 2: add bar option
+```
+
+Squashing commits during discussion of the pull request is almost always unnecessary, and makes it
```
Comment (erickzhao) [inline CONTRIBUTING.md]: Oops I've been squashing a lot of PRs... 😅

Comment (malept) [reply CONTRIBUTING.md]: That is highly opinionated and open for discussion :smile:

Comment (erickzhao) [reply CONTRIBUTING.md]: I do agree with the general spirit, though. I find it a shame when a PR makes a few different logical changes but gets squashed and the resulting commit history is unclear!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+- [Pull requests](#filing-pull-requests)
+
+We strongly suggest that before filing an issue, you search through the existing issues to see
+if it has already been filed by someone else.
+
+This project is a part of the Electron ecosystem. As such, all contributions to this project follow
+[Electron's code of conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md)
+where appropriate.
+
+## Contribution suggestions
```
Comment (felixrieseberg) [inline CONTRIBUTING.md]:
```suggestion
## Contribution Suggestions
```

(I don't care about title case or not, but I have a weird thing for consistency)

File: CONTRIBUTING.md (hunk 1 above)
Comment (felixrieseberg) [reply CONTRIBUTING.md]: My two cents: Let's cut this whole part. I don't _really_ care what people do as long as they contribute at all. Fiddle isn't exactly drowning in people doing work, so I'd rather lower the barrier to entry.

Just like y'all I don't have very strong opinions though...

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+```
+
+Squashing commits during discussion of the pull request is almost always unnecessary, and makes it
+more difficult for both the submitters and reviewers to understand what changed in between comments.
+However, rebasing is encouraged when practical, particularly when there's a merge conflict.
+
+If you are continuing the work of another person's PR and need to rebase/squash, please retain the
+attribution of the original author(s) and continue the work in subsequent commits.
+
+### Running Fiddle from source
```
Comment (felixrieseberg) [inline CONTRIBUTING.md]:
```suggestion
### Running Fiddle From Source
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+```sh
+git clone https://github.com/electron/fiddle
+cd fiddle
+npm install
+npm start
+```
+
+### Running tests
```
Comment (felixrieseberg) [inline CONTRIBUTING.md]:
```suggestion
### Running Tests
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+```sh
+npm test
+```
+
+## Release process
```
Comment (felixrieseberg) [inline CONTRIBUTING.md]:
```suggestion
## Release Process
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+<!-- TODO @felix add your release process here 😁 -->
```
Comment (felixrieseberg) [inline CONTRIBUTING.md]: Good call, I should do that 😅 We merge this first and then add it?

**Comment by felixrieseberg** [comment]
Thank you for this!

File: CONTRIBUTING.md (hunk 1 above)
Comment (malept) [reply CONTRIBUTING.md]: I'll get rid of it.

File: CONTRIBUTING.md (hunk 6 above)
Comment (malept) [reply CONTRIBUTING.md]: Sounds good to me.

**Pull request merged**